tests/fixtures/ember-container-query-glint/output/src/components/widgets/widget-1.ts

```diff
@@ -2,10 +2,6 @@ import templateOnlyComponent from '@ember/component/template-only';
 
 interface WidgetsWidget1Signature {
   Args: {};
-  Blocks: {
-    default: [];
-  };
-  Element: null;
```

Comment:
I mistakenly thought that, for components that have the correct Glint signature, the codemod would remove `Blocks` and `Element`.

It turned out, [the input project had an empty signature](https://github.com/ijlee2/ember-codemod-args-to-signature/blob/0.1.0/tests/fixtures/ember-container-query-glint/input/src/components/widgets/widget-1.ts#L3). So "adding `Args` only" is correct.